main.py

```
@@ -1107,6 +1509,285 @@ async def _sync_process(update: Update, context: ContextTypes.DEFAULT_TYPE, pack
+    return await update.effective_message.reply_text(
+        "🧙 <i>working on it…</i>",
+        parse_mode="HTML"
+    )
+
+
+async def delete_working_animation(bot, chat_id: int, message_id: int):
+    """Delete the working-animation placeholder sent by send_working_animation."""
+    try:
+        await bot.delete_message(chat_id=chat_id, message_id=message_id)
+    except Exception:
+        pass  # already deleted or not found — safe to ignore
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)